extensions/oblique-strategies/src/generate-new-oblique-strategy.tsx

```diff
@@ -0,0 +1,22 @@
+import { Action, ActionPanel, Detail } from "@raycast/api";
+import { useAI } from "@raycast/utils";
+
+const OBLIQUE_PROMPT =
+  "Create a new oblique strategy in the vain of Brian Eno and Peter Schmidt. Only reply with the short suggestion, nothing else.";
+
+export default function Command() {
+  const { data, isLoading } = useAI(OBLIQUE_PROMPT);
+
```

Comment:
Should probably check if the user has [access to AI](https://developers.raycast.com/api-reference/environment#example-1)

Reply:
@pernielsentikaer I added the recommended check to only execute the useAI hook, if AI access is available. IMO the experience is nicer without it though, since I previously got directed to a screen from Raycast which offers a trial subscription. Now it will just show the error message. Unfortunately I can't really test if the prompt works well without the subscription, so I could also just remove the AI command if it causes problems.